tests/fixtures/backends.py

```diff
@@ -36,11 +36,11 @@ class NamedClassEnum(Enum):
 
 @pytest.fixture
 def es():
-    """Create / delete an ElasticSearch test index and yield an instanciated client"""
+    """Create / delete an ElasticSearch test index and yield an instantiated client"""
     # pylint: disable=invalid-name
 
     client = Elasticsearch(ES_TEST_HOSTS)
-    client.indices.create(index=ES_TEST_INDEX, ignore=400)
```

Comment:
pylint is complaining here about: `Unexpected keyword argument 'ignore' in method call (unexpected-keyword-arg)`.
But this is only a pylint issue.
I propose to remove this argument as the tests pass without it.
Would it be better if we keep the `ignore` and add a `pylint: disable=unexpected-keyword-arg` ?

Reply:
:thinking: no feel free to remove non-required argument. Thanks!